### PR TITLE
[PR1] DEV-13 Added streaming routines

### PR DESCRIPTION
**Pull request by smorsua**
This functions handle reading / writing to the websocket connections. The data is sent / received from the exterior using channels.

### CODE REVIEW

File: src/DataTransfer/domain/measurement/value/bool.go [hunk 1]
```diff
@@ -6,7 +6,23 @@ import (
...
 type Bool bool
 
-func (b Bool) ToDisplayString() string {
+func (b Bool) ToDisplayUnitsString() string {
+	return b.toString()
+}
+
+func (b *Bool) GetPodUnits() string {
+	return ""
+}
```
Comment (lipezaballa) [inline src/DataTransfer/domain/measurement/value/bool.go:15]: Is it correct that if this type of data does not need this method, implement it in blank? Is it not better to start all by default in blank and depending on the type use only the necessary ones?

File: src/Shared/StreamingHandlers/streaming.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+			timeout := time.After(time.Millisecond * 20)
+		loop:
+			for {
+				select {
+				case packet := <-packetChannel:
+					adapter := newPacketWebAdapter(packet)
+					packetWebAdapterBuf = append(packetWebAdapterBuf, adapter)
+					if len(packetWebAdapterBuf) == 100 {
+						ws.WriteJSON(packetWebAdapterBuf)
+						break loop
```
Comment (lipezaballa) [inline src/Shared/StreamingHandlers/streaming.go:23]: Run it only for the last packet, either here or with the timeout? Why?

File: src/Shared/StreamingHandlers/test.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
+package streaming
```
Comment (jmaralo) [inline src/Shared/StreamingHandlers/test.go:1]: remove commented code

File: src/DataTransfer/domain/measurement/value/number/Number.go [hunk 4]
```diff
@@ -14,7 +14,20 @@ func NewNumber(podUnitString string, displayUnitString string) *Number {
 	return &Number{value: 0, podUnits: podUnits, displayUnits: displayUnits}
 }
 
-func (i Number) ToDisplayString() string {
+func (n *Number) GetPodUnits() string {
```
Comment (jmaralo) [inline src/DataTransfer/domain/measurement/value/number/Number.go:17]: why does this function receive a pointer?

**Review by jmaralo — CHANGES_REQUESTED** [review]
just a few things here and there

File: src/DataTransfer/domain/measurement/value/bool.go (hunk 1 above)
Comment (smorsua) [reply src/DataTransfer/domain/measurement/value/bool.go:15]: Necesita implementarlo porque necesita implementar la interface Value. A cada tipo hay que implementar la función por separado, no puedes implementárselo a varios de golpe

**Review by jmaralo — DISMISSED** [review]

**Review by lipezaballa — APPROVED** [review]

**Review by jmaralo — APPROVED** [review]

**Review by lipezaballa — APPROVED** [review]

**Pull request merged**